src/main/java/de/tum/in/www1/artemis/web/rest/ParticipationResource.java

```diff
@@ -306,7 +306,7 @@ private void addLatestResultToParticipation(Participation participation) {
         if (!courseService.userHasAtLeastTAPermissions(course)) {
             throw new AccessForbiddenException("You are not allowed to access this resource");
         }
-        List<Participation> participations = participationService.findByCourseIdWithRelevantResults(courseId);
+        List<Participation> participations = participationService.findByCourseIdWithRelevantResults(courseId, false);
```

Comment:
This is a very performance heavy operation. In a course with 2000 participants and 80 exercises, this loads a lot of data. And now it gets slower. Can you please split the 2 cases, because here we don't need the assessor?

Reply:
Done